src/main/java/io/antmedia/statistic/StatsCollector.java

```diff
@@ -367,6 +377,18 @@ public static JsonObject getSysteMemoryInfoJSObject() {
 		jsonObject.addProperty(IN_USE_SWAP_SPACE, SystemUtils.osInUseSwapSpace());
 		return jsonObject;
 	}
+	
+	public static JsonObject getRAMMemoryInfoJSObject() {
+		JsonObject jsonObject = new JsonObject();
+		
+		long totalPhysicalBytes = Pointer.totalPhysicalBytes();
+		long physicalBytes = Pointer.availablePhysicalBytes();
+		
+		jsonObject.addProperty(FREE_NATIVE_MEMORY, physicalBytes);
```

Comment:
Are you sure `Pointer.totalPhysicalBytes` and `Pointer.availablePhysicalBytes`  are the ones we need to monitor?

Because in enoughResource method we use the below methods
```
long maxPhysicalBytes = Pointer.maxPhysicalBytes();
long physicalBytes = Pointer.physicalBytes();
```